Gemfile

```diff
@@ -10,8 +10,8 @@ group :test, :development do
   gem 'rspec', '~> 3.1'
 
   unless ENV['NO_ACTIVERECORD']
-    gem 'activerecord', '>= 3.2.3', '< 4.3.0'
-    gem 'activerecord-oracle_enhanced-adapter', '>= 1.4.1', '< 1.7.0'
+    gem 'activerecord', '>= 3.2.3', '< 5.1.0'
```

Comment:
Yes. [Ruby 2.2.2+ required](http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#ruby-2-2-2-required) to run Rails 5.0. I am not familiar with limiting ruby version per Rails version, though.